decidim-core/db/migrate/20170809084005_add_scope_enabled_to_participatory_processes.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddScopeEnabledToParticipatoryProcesses < ActiveRecord::Migration[5.1]
+  def change
+    add_column :decidim_participatory_processes, :scope_enabled, :boolean, null: false, default: true
```

Comment:
Also, note that you are using the `has_scopes?` method (notice the plural), so that's for consistency too